p1/autograde.py

```diff
@@ -39,7 +39,7 @@ def compose_test():
     with open("compose.txt") as f:
         if "v2.20.2" in f.read():
```

Comment:
Do you intend to update these as well? In fact, @tylerharter, should we get rid of this test completely since we did not specify that in spec?

Reply:
Fixed